Apps/Sandcastle/gallery/CZML Polygon - Intervals, Availability.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
I'm actually not seeing that flicker?
![out](https://cloud.githubusercontent.com/assets/6036502/10217613/051dea22-67ff-11e5-805a-5fbf0700a5f1.gif)